tests/functional/s3/test_presign_command.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import datetime
+
+from botocore.compat import urlsplit
+from awscli.testutils import BaseAWSCommandParamsTest, mock, temporary_file
+from awscli.testutils import create_clidriver
+
+
+# Values used to fix time.time() and datetime.datetime.utcnow()
+# so we know the exact values of the signatures generated.
+FROZEN_TIMESTAMP = 1471305652
+DEFAULT_EXPIRES = 3600
+FROZEN_TIME = mock.Mock(return_value=FROZEN_TIMESTAMP)
+FROZEN_DATETIME = mock.Mock(
+    return_value=datetime.datetime(2016, 8, 18, 14, 33, 3, 0))
+
+
+class TestPresignCommand(BaseAWSCommandParamsTest):
+
+    maxDiff = None
```

Comment:
`BaseAWSCommandParamsTest` already [sets this](https://github.com/jamesls/aws-cli/blob/d34b89a0d32f36ed4948b19afe4dd00c6c95ab3c/awscli/testutils.py#L306), does it not carry through?